scripts/zones/Lufaise_Meadows/mobs/Flockbock.lua

```diff
@@ -1,11 +1,45 @@
 -----------------------------------
 -- Area: Lufaise Meadows
 --   NM: Flockbock
+-- https://www.bg-wiki.com/ffxi/Flockbock
+-- TODO: NM has several possible spawn locations
+-----------------------------------
+mixins = { require('scripts/mixins/rage') }
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:addMod(xi.mod.BINDRES,      100)
+    mob:addMod(xi.mod.SLEEPRES,     100)
+    mob:addMod(xi.mod.GRAVITYRES,   100)
+    mob:addMod(xi.mod.SILENCERES,   100)
+
+    -- Has enhanced double attack, movement speed, and uses WS more often than normal mobs.
+    mob:setMod(xi.mod.DOUBLE_ATTACK, 50)
+    mob:setMod(xi.mod.MOVE,          25)
```

Comment:
how did you confirm this monsters move speed, and how did you confirm +25 mod move would bring it to that speed??

Reply:
gonna add to this that using `mod_mov` in a mob script is extremely prone to being problematic. Its better to cap the real speed in retail (gonna go look if mob is up right now) and set it as that speed in either its family or if that is not possible, the `setSpeed()` function.

Some mobs also have a "chase speed" which LSB does not yet have a method for. Not all mobs do that.

Reply:
I had Spicy take a video for me, didn't bother posting it in the captures discord because Silknoz already had one, but I'm happy to provide it if you like ... also happy to use setSpeed instead of the mod.